### PR TITLE
added text prop for allowing user to put text below loader

### PR DESCRIPTION
**Pull request by Enigma10**

### CODE REVIEW

File: example/App.vue [hunk 1]
```diff
@@ -66,6 +66,7 @@
     <div class="btn-toggle">
       <button @click="activateFullScreenGreenBg()">Full Screen (Green background color)</button>
     </div>
+    <input class="btn-toggle" type="text" v-model="text" placeholder="custom text "></input>
```
Comment (biigpongsatorn) [inline example/App.vue]: - Please remove a space after `custom text`.
- Input tag should be `<input/>`, not `<input></input>`.

File: src/index.vue [hunk 2]
```diff
@@ -10,6 +10,7 @@
         <slot name="default">
           <component :is="spinner" :color="color" :size="`${size}px`" :duration="`${duration}s`"></component>
         </slot>
+        <div :style="{ color }" >{{text}}</div>
```
Comment (biigpongsatorn) [inline src/index.vue]: Please use `v-if` to check `text` is existing and please add space inside `{{text}}` like this `{{ text }}`

**Comment by Enigma10** [comment]
Okay will make it :)

**Comment by Enigma10** [comment]
@biigpongsatorn Please check once :)

**Comment by biigpongsatorn** [comment]
@Enigma10 OK, Let me check.

**Comment by biigpongsatorn** [comment]
@Enigma10  I has published your feature to version `1.0.8`.

**Pull request merged**